sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -292,7 +294,7 @@ export interface CreateKeyOptions extends coreHttp.OperationOptions {
 /**
  * @interface
  * An interface representing the optional parameters that can be
- * passed to {@link beginDeleteKey}
+ * passed to {@link beginDeleteKey} and {@link beginRecoverDeletedKey}
```

Comment:
Is "and" better or "or"....
We should get a vote on this :)

Reply:
depends on whether we're reading this as an or or as an exclusive or 😝 and seems more truthful since both functions take the same argument (not one or the other).